core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -145,22 +223,28 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
       // directly stored in the customizationArgs.imageAndRegion.value
       // .imagePath
       if (state.interaction.id === INTERACTION_TYPE_IMAGE_CLICK_INPUT) {
-        filenamesInState.push(
+        var filepathObject = (
           state.interaction.customizationArgs.imageAndRegions.value.imagePath);
+        if (!filepathObject.width) {
+          var filename = filepathObject;
+          filepathObject = {
+            name: filename,
+            width: 500,
+            height: 200
+          };
+        }
```

Comment:
I think we don't need this here, since we only use the name later.

Reply:
We are using it. The _getImageDimensionsFromFilepathValue gets the dimensions of the images and we later on use these for the loading indicator.

Reply:
Oh! got it. Will do the same.

Reply:
@ishucr7 ping!

Reply:
Sorry for the previous comment.
No, we use this to get the dimensions if the images. Check line 172 in function `_getImageDimensionsFromFilepathValue` . For the images already there in oppia we need to assign a default dimension, and this is what we need this for.

Reply:
Yeah,  understood it.